apps/web-giddh/src/app/subscription/subscription.component.ts

```diff
@@ -204,10 +204,12 @@ export class SubscriptionComponent implements OnInit, OnDestroy {
             }
         });
 
-        this.store.pipe(select(state => state.session.activeCompany), takeUntil(this.destroyed$)).subscribe(activeCompany => {
+        this.store.pipe(select(profile => profile.settings.profile), takeUntil(this.destroyed$)).subscribe((activeCompany) => {
             if (activeCompany) {
+                this.selectedCompany = null;
```

Comment:
what's this????

Reply:
@ravinderwalkover sir, for refresh proper response

Reply:
Fix this